lib/features/login/domain/usecases/get_token_oidc_interactor.dart

```diff
@@ -46,6 +47,10 @@ class GetTokenOIDCInteractor {
       yield Right<Failure, Success>(GetTokenOIDCSuccess(tokenOIDC, config));
     } catch (e) {
       logError('GetTokenOIDCInteractor::execute(): $e');
+      if (NoSuitableBrowserForOIDCException.verifyNoSuitableBrowserOIDC(e)) {
+        yield Left<Failure, Success>(GetTokenOIDCFailure(NoSuitableBrowserForOIDCException()));
+        return;
+      }
       yield Left<Failure, Success>(GetTokenOIDCFailure(e));
     }
```

Comment:
```suggestion
    } on PlatformException catch (e) {
      logError('GetTokenOIDCInteractor::execute():PlatformException: $e');
      if (exception.code == noBrowserAvailableCode) {
        yield Left<Failure, Success>(GetTokenOIDCFailure(NoSuitableBrowserForOIDCException()));
      } else {
        yield Left<Failure, Success>(GetTokenOIDCFailure(e));
      }
    } catch (e) {
      logError('GetTokenOIDCInteractor::execute(): $e');
      yield Left<Failure, Success>(GetTokenOIDCFailure(e));
    }
```

Reply:
IMO exception should not contain business logic

Reply:
no, it is not a business logic, it is a mapping from external exception to business exception